youtube_dl/extractor/facebook.py

```diff
@@ -423,16 +452,35 @@ def extract_from_jsmods_instances(js_data):
             video_title = 'Facebook video #%s' % video_id
         uploader = clean_html(get_element_by_id(
             'fbPhotoPageAuthorName', webpage)) or self._search_regex(
-            r'ownerName\s*:\s*"([^"]+)"', webpage, 'uploader',
-            fatal=False) or self._og_search_title(webpage, fatal=False)
+            r'ownerName\s*:\s*"([^"]+)"', webpage, 'uploader',default=None) or \
+                   self._og_search_title(webpage, default=None) or self._search_regex(
+                        r'\"ownerName\":"(.+?)"', tahoe_secondary_data,
```

Comment:
This breaks if `tahoe_secondary_data` request has failed.

Reply:
Thanks. I added empty string as default.